joblib/test/test_my_exceptions.py

```diff
@@ -2,6 +2,7 @@
 Test my automatically generate exceptions
 """
 from joblib import my_exceptions
+from _pytest.assertion.util import BuiltinAssertionError
```

Comment:
@lesteve I fixed it just to mkae sure it doesn;t happen anywhere else. This happens on my machine:

![image](https://cloud.githubusercontent.com/assets/10494087/20793653/1368f94c-b7ee-11e6-89a1-417b21140739.png)

I came to know by this that `pytest` has its own variant of `AssertionError`.